Packs/FireMonSecurityManager/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "FireMon Security Manager",
+    "description": "Creates a Policy Planner Ticket in Policy Planner Application and Validates Pre Changes assessments for new requirement.",
+    "support": "xsoar",
```

Comment:
The support is parter right? 
Need to change the support, url and author to contain details regarding the partner.